lib/statistics.js

```diff
@@ -1,43 +1,33 @@
 'use strict';
 
-const Stats = require('fast-stats').Stats;
-
-function percentileName(percentile) {
-  if (percentile === 0) {
-    return 'min';
-  } else if (percentile === 100) {
-    return 'max';
-  } else if (percentile === 50) {
-    return 'median';
-  }
-  return `p${String(percentile).replace('.', '_')}`;
-}
-
 class Statistics {
   constructor() {
-    this.stats = new Stats();
+    this.values = [];
   }
 
   add(value) {
-    this.stats.push(value);
+    this.values.push(value);
     return this;
   }
 
-  summarize(options) {
-    if (isNaN(this.stats.amean())) {
+  summarize() {
+    const values = this.values;
+    // keeping backward compability
+    if (values.length === 0) {
       return undefined;
     }
 
-    options = options || {};
-    const percentiles = options.percentiles || [0, 10, 50, 90, 99, 100];
-    const decimals = options.decimals || 0;
-    const data = {};
-    const stats = this.stats;
+    values.sort((a, b) => a - b);
+
+    const data = {
+      min: Number(values[0]).toFixed(0),
+      max: Number(values[values.length - 1]).toFixed(0),
+    };
 
-    percentiles.forEach((p) => {
-      const name = percentileName(p);
-      data[name] = Number(stats.percentile(p).toFixed(decimals));
-    });
+    const middle = Math.floor(values.length / 2);
+    const isEven = values.length % 2 === 0;
+    data.median = isEven ?
```

Comment:
In the !isEven case, this doesn't call .toFixed(0) on the value. Adding that, I think there's a little to much going on in one line. Better to break it up to an if/else.

One suggestion, to make it more visually obvious that all values are treated equally, is to set it up as:

```
const min = values[0];
const max = values[values.length - 1];

//... calculate median

return {
   min: Number(min).toFixed(0),
   median: Number(median).toFixed(0),
   max: Number(max).toFixed(0),
};
```